docs/reference-metrics-opentelemetry.md

```diff
@@ -8,6 +8,153 @@ Pulsar exposes the following OpenTelemetry metrics.
 
 ## Broker
 
```

Comment:
Can we clarify the OTel attributes related to partitioned or non-partitioned topics?